v2/features/stats/chain/AssetContainer.tsx

```diff
@@ -0,0 +1,109 @@
+import { Link, Text } from "@zeit-ui/react";
```

Comment:
Let's not use this lib anymore

Reply:
deleted this whole file, it was no longer being used